eset-protect-enterprise/info.json

```diff
@@ -0,0 +1,672 @@
+          "tooltip": "Specify the device UUID to end isolation from network."
+        },
+        {
+          "title": "Device Group UUID",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "device_group_uuid",
+          "description": "Specify the device group UUID to end isolation from network.",
+          "tooltip": "Specify the device group UUID to end isolation from network."
+        },
+        {
+          "title": "Task Expire Time in minutes",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "task_expire_time",
+          "description": "Specify the minutes so that this task will get expired after that.",
+          "tooltip": "Specify the minutes so that this task will get expired after that.",
+          "value": 10
+        },
+        {
+          "title": "Task Display Name",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "task_display_name",
+          "description": "Specify the the name for new created task.",
+          "tooltip": "Specify the the name for new created task."
+        },
+        {
+          "title": "Task Description",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "task_description",
+          "description": "Specify the the description for new created task.",
+          "tooltip": "Specify the the description for new created task."
+        }
+      ],
+      "enabled": true,
+      "output_schema": {}
+    },
+    {
+      "operation": "block_executables",
+      "title": "Block Executables",
+      "description": "Block executables by provided UUID",
+      "category": "containment",
+      "annotation": "block_executables",
+      "parameters": [
+        {
+          "title": "Application Management URL",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "server_url",
+          "description": "Specify the server URL for Application Management.",
+          "tooltip": "Specify the server URL for Application Management.",
+          "value": "https://eu.application-management.eset.systems"
+        },
+        {
+          "title": "Executable UUID",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "executableUuid",
+          "description": "Specify the Executable UUID to the blocked.",
+          "tooltip": "Specify the Executable UUID to the blocked."
+        },
+        {
+          "title": "Request Body",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "json_data",
+          "description": "Request body.",
```

Comment:
please add detailed description

Reply:
ESET itself has not provided much information about this parameter, and we do not have a setup to test this option. Therefore, I have used the same parameter name and description as provided in the ESET Swagger file, so that users can relate the parameter to the connector actions parameter. 

<img width="1563" alt="image" src="https://github.com/fortinet-fortisoar/connector-eset-protect-enterprise/assets/126162532/aa00dc45-bb9d-40d7-a569-8ae6822f270f">